common/go-utils/src/bin/update_inventory.rs

```diff
@@ -17,16 +17,10 @@ fn main() {
         process::exit(4);
     });
 
-    let mut inventory = Inventory {
+    let inventory = Inventory {
         artifacts: remote_artifacts,
     };
 
-    // Sort artifacts in reverse semver order, to make it easier to resolve
-    // to the most recent version for a semver constraint.
-    inventory
-        .artifacts
-        .sort_by(|b, a| a.semantic_version.cmp(&b.semantic_version));
```

Comment:
why was the sorting removed here?

Reply:
It isn't necessary anymore - the inventory is being written in the same order as we receive it from upstream (as previous PRs removed the majority of logic contained in this function, in favor of rewriting the entire `inventory.toml` every time).

Turns out the go.dev/dl feed (unsurprisingly) return artifacts in the same order we used to explicitly sort them by here :)

Reply:
i figured it had to do with the feed already being ordered. thanks for confirming.